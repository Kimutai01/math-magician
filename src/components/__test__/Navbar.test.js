import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from '../Navbar';

it('testing the navbar', () => {
  const TREE = renderer.create(
    <Router>
      <Nav />
    </Router>
  ).toJSON;
  expect(TREE).toMatchSnapshot();
});
