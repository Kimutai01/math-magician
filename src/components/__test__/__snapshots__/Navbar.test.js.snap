// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the navbar 1`] = `[Function]`;
